src/v2/service/addon.service.ts

```diff
@@ -16,6 +16,22 @@ import {
 } from "../interfaces/addons";
 import AddonFirestormRepository from "../repository/addon.repository";
 import { discordEmbed } from "../tools/discordEmbed";
+import { fileTypeFromBuffer, MimeType } from "file-type";
+
+const HEADER_MIME_TYPES: MimeType[] = [
+	"image/jpeg"
+];
+
+const SCREENSHOT_MIME_TYPES: MimeType[] =  [
+	"image/jpeg"
+];
+
+async function verifyFileType(buffer: Buffer, mime_types_accepted: Array<MimeType>) {
```

Comment:
I feel like this should be part of the service class as a method (bit cleaner since that way there's only one declaration)